.devcontainer/Dockerfile

```diff
@@ -0,0 +1,116 @@
+FROM debian
+
+RUN apt update
+
+# Create sudo user "dev"
+ARG USERNAME=dev
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Create the `dev` user
+# https://code.visualstudio.com/remote/advancedcontainers/add-nonroot-user
+RUN groupadd --gid $USER_GID $USERNAME \
+    && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
+    && apt-get update \
+    && apt-get install -y sudo \
+    && echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME \
+    && chmod 0440 /etc/sudoers.d/$USERNAME
+USER $USERNAME
+
+# zcashd dependencies
+RUN sudo apt install -y git wget build-essential libncurses5 autotools-dev \
+    automake libtool bsdmainutils pkg-config curl
+
+# Install rust
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
```

Comment:
Have you considered something like this:

https://github.com/zancas/zecwallet-mobile/blob/8eee2ceb0abf4f616f928f40fdedb296a3854178/rust/android/Dockerfile#L34

It uses the flags chosen by the rust language default install invocation, and since it uses `-o` it drops a copy onto the docker image layer.
@dannasessha pushed for the `-o` for auditability.

Reply:
the idea is that if sh.rustup.rs is corrupted that there will be a way to see what happened after the fact?